panel/pane/holoviews.py

```diff
@@ -505,6 +505,8 @@ def _render(self, doc, comm, root):
                 Store.current_backend = loaded_backend
             backend = self.backend or Store.current_backend
             renderer = Store.renderers[backend]
+        if state.browser_info and 'webgl' in renderer.param:
+            renderer.webgl = state.browser_info.webgl
```

Comment:
I don't think it is Panel job to do this.

Reply:
Can see arguments both ways. I'd say this is the best place to do it though.